bindings/rust/bench/src/s2n_tls.rs

```diff
@@ -26,10 +26,8 @@ use std::{
 pub struct S2NHarness {
     // UnsafeCell is needed b/c client and server share *mut to IO buffers
     // Pin<Box<T>> is to ensure long-term *mut to IO buffers remain valid
-    client_to_server_buf: Pin<Box<UnsafeCell<VecDeque<u8>>>>,
-    server_to_client_buf: Pin<Box<UnsafeCell<VecDeque<u8>>>>,
-    client_config: Config,
-    server_config: Config,
+    _client_to_server_buf: Pin<Box<UnsafeCell<VecDeque<u8>>>>,
```

Comment:
Why is this begin with an underscore? I thought that was only for fields that weren't actually used or something.

Reply:
The underscores are to mark the fields that are private and not part of the public api (ex. it won't appear in the docs).

Reply:
I don't think we need to worry about doc level visibility stuff for a benchmarking tool. And I agree with Madeline that we only want to use `_` for things that are unused, so if we're using this field we should remove the `_` prefix.